ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/maven/tasks/main.yml

```diff
@@ -20,11 +20,7 @@
   tags: maven
 
 - name: Download Apache Maven v3.6.3 (Solaris)
-  get_url:
-   url: https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
-   dest: /tmp/
-   checksum: sha512:c35a1803a6e70a126e80b2b3ae33eed961f83ed74d18fcd16909b2d44d7dada3203f1ffe726c17ef8dcca2dcaa9fca676987befeadc9b9f759967a8cb77181c0
-   validate_certs: False
+  command: wget https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz -O /tmp/apache-maven-3.6.3-bin.tar.gz
```

Comment:
Can we do a SHA check afterwards? Otherwise it looks a bit odd to be removing the checksum verification here.

Reply:
I switched to wget because of the usual SSL issues that we see on Solaris. I'm happy to add checksum validation back but there's a whole set of roles doing the same thing so this should likely be a follow up PR